### ISSUE TITLE
Accessibility Statement

### ISSUE THREAD

**Issue by declanbyrd** [declanbyrd/declanbyrd.co.uk#12]
I would like to conduct a full accessibility audit and display on the site what has been done to make it accessible. Failures should be fixed if found.
